docs/preview/actions-runner-controller-2/README.md

```diff
@@ -0,0 +1,129 @@
+
+    ```bash
+    NAMESPACE="arc-systems"
+    helm install arc \
+        --namespace "${NAMESPACE}" \
+        --create-namespace \
+        oci://ghcr.io/actions/actions-runner-controller-charts/actions-runner-controller-2 \
+        --version 0.1.0
+    ```
+
+1. Generate a Personal Access Token (PAT) or create and install a GitHub App. See [Creating a personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) and [Creating a GitHub App](https://docs.github.com/en/developers/apps/creating-a-github-app).
+
+1. You're ready to install the autoscaling runner set. For additional configuration options, see [values.yaml](https://github.com/actions/actions-runner-controller/blob/master/charts/auto-scaling-runner-set/values.yaml)
+
+    ```bash
+    # Using a Personal Access Token (PAT)
+    $ helm install arc-runner-set \
+        --namespace "${NAMESPACE}" \
+        --create-namespace \
+        --set githubConfigUrl="https://github.com/<your_enterprise/org/repo>" \
+        --set githubConfigSecret.github_token="<PAT>" \
+        --set 
+        oci://ghcr.io/actions/actions-runner-controller-charts/auto-scaling-runner-set --version 0.1.0
+    
+    # Using a GitHub App
+    $ helm install arc-runner-set \
+        --namespace "${NAMESPACE}" \
+        --create-namespace \
+        --set githubConfigUrl="https://github.com/<your_enterprise/org/repo>" \
+        --set githubConfigSecret.github_app_id="<GITHUB_APP_ID>" \
+        --set githubConfigSecret.github_app_installation_id="<GITHUB_APP_INSTALLATION_ID>" \
+        --set githubConfigSecret.github_app_private_key="<GITHUB_APP_PRIVATE_KEY>" \
+        oci://ghcr.io/actions/actions-runner-controller-charts/auto-scaling-runner-set --version 0.1.0
+    ```
+
+1. Check your installation. If everything went well, you should see the following:
+
+    ```bash
+    $ helm list -n "${NAMESPACE}"
+
+    NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                                    APP VERSION
+    arc             arc-systems     1               2023-01-18 10:03:36.610534934 +0000 UTC deployed        actions-runner-controller-2-0.1.0        preview    
+    arc-runner-set  arc-systems     1               2023-01-18 10:20:14.795285645 +0000 UTC deployed        auto-scaling-runner-set-0.1.0            0.1.0 
+    ```
+
+    ```bash
+    $ kubectl get pods -n "${NAMESPACE}"
+
+    NAME                                              READY   STATUS    RESTARTS   AGE
+    arc-actions-runner-controller-2-8c74b6f95-gr7zr   1/1     Running   0          20m
+    arc-runner-set-6cd58d58-listener                  1/1     Running   0          21s
+    ```
+
+1. In your repository, create a simple test workflow as follows. The `runs-on` value should match the helm installation name you used in the previous step.
```

Comment:
Should we specify that the repo has to have access to the scale set or do we assume we're doing everything at repo level for this guide?

Reply:
hmm it should be implied at this point, I'm not sure it's worth the extra hand-holding here